### PR TITLE
Catalog theming

### PR DESCRIPTION
**Pull request by mohammadcazig**
OnSecondaryColor should also change when Secondary Color changes on global theming,
BottomAppBar FAB should be themed according to global theming, (https://www.pivotaltracker.com/story/show/157299010)
FeatureHighlight should be theming according to global theming, (https://www.pivotaltracker.com/story/show/157299102)
FAB button in custom shapes buttons example should be themed according to global theming. (https://www.pivotaltracker.com/story/show/157298987)
screen shot 2018-05-04 at 2 25 57 pm
screen shot 2018-05-04 at 2 26 07 pm
screen shot 2018-05-04 at 2 26 21 pm
screen shot 2018-05-04 at 2 26 24 pm
screen shot 2018-05-04 at 2 26 28 pm

### CODE REVIEW

File: components/BottomAppBar/examples/BottomAppBarTypicalUseExample.m [hunk 1]
```diff
@@ -17,22 +17,28 @@
 #import <UIKit/UIKit.h>
 
 #import "MaterialBottomAppBar.h"
-#import "MaterialButtons+ColorThemer.h"
+#import "MaterialButtons+buttonThemer.h"
```
Comment (romoore) [inline components/BottomAppBar/examples/BottomAppBarTypicalUseExample.m]: Capitalization?

File: components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseViewController.m [hunk 2]
```diff
@@ -54,6 +38,13 @@ - (void)layoutBottomAppBar {
   _bottomBarView.frame = bottomBarViewFrame;
 }
 
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  [self commonMDCBottomAppBarViewControllerSetup];
+
```
Comment (romoore) [inline components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseViewController.m]: Some extra newlines around here.

File: components/FeatureHighlight/examples/FeatureHighlightShownViewExample.m [hunk 3]
```diff
@@ -15,20 +15,28 @@
...
 #import "MaterialFeatureHighlight+FeatureHighlightAccessibilityMutator.h"
+#import "MaterialFeatureHighlight+TypographyThemer.h"
 #import "supplemental/FeatureHighlightExampleSupplemental.h"
 
 @implementation FeatureHighlightShownViewExample
 
 - (void)didTapButton:(id)sender {
-  MDCFloatingButton *fab =
-      [MDCFloatingButton floatingButtonWithShape:MDCFloatingButtonShapeDefault];
+
```
Comment (romoore) [inline components/FeatureHighlight/examples/FeatureHighlightShownViewExample.m]: Extra newlines.

File: components/FeatureHighlight/examples/supplemental/FeatureHighlightExampleSupplemental.h [hunk 4]
```diff
@@ -31,6 +31,9 @@
 
 @interface FeatureHighlightColorExample : MDCCollectionViewController
 @property(nonatomic) NSArray *colors;
+@property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
+@property(nonatomic, strong) MDCTypographyScheme *typographyScheme;
+
```
Comment (romoore) [inline components/FeatureHighlight/examples/supplemental/FeatureHighlightExampleSupplemental.h]: Extra newline.

**Review by romoore — APPROVED** [review]
Please use one PR/one issue when possible.

File: components/FeatureHighlight/examples/FeatureHighlightShownViewExample.m (hunk 3 above)
Comment (mohammadcazig) [reply components/FeatureHighlight/examples/FeatureHighlightShownViewExample.m]: Done.

**Review by jverkoey — CHANGES_REQUESTED** [review]
Please break this in to separate PRs.

**Comment by mohammadcazig** [comment]
Made separate PR.

**Pull request closed without merging**